src/raft.h

```diff
@@ -9,38 +9,44 @@ namespace cura {
 class Raft
 {
 public:
-    static void generate(SliceDataStorage& storage, int distance);
+    static void generate(SliceDataStorage& storage, coord_t distance);
 
     /*!
-     * Get the height difference between the raft and the bottom of layer 1.
+     * \brief Get the height difference between the raft and the bottom of
+     * layer 1.
      * 
-     * This is used for the filler layers because they don't use the layer_0_z_overlap
+     * This is used for the filler layers because they don't use the
+     * layer_0_z_overlap.
      */
-    static int getZdiffBetweenRaftAndLayer1(const SliceDataStorage& storage);
+    static coord_t getZdiffBetweenRaftAndLayer1();
```

Comment:
I consider that outside the scope of this refactor.

Reply:
Mmmm, I know, but since you changed this line, it's good to take advantage and fix the things that we see are wrong. Following your statement why are you fixing code style in parts that you didn't touch?

OK, next time.